polynote-kernel/src/main/scala/polynote/messages/Messages.scala

```diff
@@ -106,14 +129,82 @@ object CellResult extends MessageCompanion[CellResult](4)
 
 final case class ContentEdit(pos: Int, length: Int, content: String) {
   def applyTo(rope: Rope): Rope = if (length > 0) rope.delete(pos, length).insertAt(pos, Rope(content)) else rope.insertAt(pos, Rope(content))
```

Comment:
What do you think about changing `length` to something like `overwrite` or `overwriteLength`?

Reply:
You're right, it's confusing... I changed it to `deleteLength` and added a doc comment clarifying the fields.

Reply:
Not super important for this PR but might consider changing some of the pattern matching extractions to match the new name (e.g., `s/otherLength/otherDelLen/` or something).

Reply:
Good point, no time like the present (done)